docs/index.md

```diff
@@ -148,11 +148,52 @@ Available via `.context_diagram` on a [`ModelObject`][capellambse.model.common.e
               <figcaption>Context diagram of educate Wizards LogicalFunction with type [LDFB]</figcaption>
           </figure>
 
-*   [`pa.PhysicalComponent`][capellambse.model.layers.pa.PhysicalComponent] (PAB)
-*   [`pa.PhysicalFunction`][capellambse.model.layers.pa.PhysicalFunction] (PDFB)
-*   [`pa.PhysicalComponent`][capellambse.model.layers.pa.PhysicalComponent] (PAB)
+-   ??? example "[`pa.PhysicalComponent`][capellambse.model.layers.pa.PhysicalComponent] (PAB)"
+
+    -   Physical Node Component
+
+            ``` py
+            import capellambse
+
+            model = capellambse.MelodyModel("tests/data/ContextDiagram.aird")
+            diag = model.by_uuid("fdb34c92-7c49-491d-bf11-dd139930786e").context_diagram
+            diag.render("svgdiagram").save(pretty=True)
+            ```
+            <figure markdown>
+                <img src="assets/images/Context of Physical Component.svg" width="1000000">
+                <figcaption>Context of Physical Component [PAB]</figcaption>
+            </figure>
+
+    -   Physical Behaviour Component
+
+            ``` py
+            import capellambse
+
+            model = capellambse.MelodyModel("tests/data/ContextDiagram.aird")
+            diag = model.by_uuid("313f48f4-fb7e-47a8-b28a-76440932fcb9").context_diagram
+            diag.render("svgdiagram").save(pretty=True)
+            ```
+            <figure markdown>
+                <img src="assets/images/Context of PC Software.svg" width="1000000">
+                <figcaption>Context diagram of PC Software [PAB]</figcaption>
+            </figure>
```

Comment:
This doesn't work. With MK Docs the indentation is very important. You can test the documentation with `mkdocs serve`, it is also hot reloading showing changes immediately saving a file. 
```suggestion
          `PhysicalNodeComponent`

          ``` py
          import capellambse

          model = capellambse.MelodyModel("tests/data/ContextDiagram.aird")
          diag = model.by_uuid("fdb34c92-7c49-491d-bf11-dd139930786e").context_diagram
          diag.render("svgdiagram").save(pretty=True)
          ```
          <figure markdown>
              <img src="assets/images/Context of Physical Component.svg" width="1000000">
              <figcaption>Context of Physical Component [PAB]</figcaption>
          </figure>

          `PhysicalBehaviourComponent`

          ``` py
          import capellambse

          model = capellambse.MelodyModel("tests/data/ContextDiagram.aird")
          diag = model.by_uuid("313f48f4-fb7e-47a8-b28a-76440932fcb9").context_diagram
          diag.render("svgdiagram").save(pretty=True)
          ```
          <figure markdown>
              <img src="assets/images/Context of PC Software.svg" width="1000000">
              <figcaption>Context diagram of PC Software [PAB]</figcaption>
          </figure>
```